auth-tokens-filter/src/main/java/com/palantir/tokens2/auth/http/BearerTokenLoggingFilter.java

```diff
@@ -79,6 +79,6 @@ private void setUnverifiedContext(ContainerRequestContext requestContext, String
     }
 
     public static String getRequestPropertyKey(String key) {
-        return "com.palantir.tokens.auth." + key;
+        return "com.palantir.tokens2.auth." + key;
```

Comment:
yeah, could go either way on this, probably would've left it unchanged, but as long as the filters are updated it's fine